packages/yoga/src/Input/web/Input.jsx

```diff
@@ -98,6 +105,8 @@ const Input = React.forwardRef(
       rightIcon,
       a11yId,
       includeAriaAttributes,
+      prependElement,
+      prependElementWidth,
```

Comment:
This prop shouldn't be necessary. The size of the prependElement should be defined by it's own component. 
Do you agree?

Reply:
The _prependElementWidth_ is needed to add a text-indent to the styled-component's input, so the text start next to the element. Without the prop, it would be necessary to add some calculation inside the existing Input.tsx to detect the width from the rendered element after the DOM render.